trendmicro_apex/plugin.spec.yaml

```diff
@@ -126,3 +718,546 @@ actions:
         description: Whether or not the action was successful
         type: boolean
         required: true
+  agent_list:
+    title: List Security Agents
```

Comment:
This action should be titled "Search Agents" and use this API endpoint to search across IPs, MAC addresses, etc.

https://beta-community-trendmicro.cs23.force.com/automationcenter/apex-central/api#tag/Security-Agents